presto-spi/src/main/java/com/facebook/presto/spi/type/UnscaledDecimal128Arithmetic.java

```diff
@@ -1686,7 +1686,8 @@ else if (high != 0x4b3b4ca85a86c47aL) {
     private static byte[] reverse(final byte[] a)
     {
         final int length = a.length;
-        for (int i = length / 2; i-- != 0; ) {
+        int i = length / 2;
```

Comment:
Sorry about that once again. I have to many context switches. And sometimes things leak.